deployments/engine/docker-compose/dm_databases_tidb_new_collation_off.yaml

```diff
@@ -0,0 +1,61 @@
+version: '2.3'
```

Comment:
Better to use a general filename, such as: `dm_database_tidb_v6.1.yaml`

Reply:
new_collation_off is the key difference of this deployment. (I'll change v6.1 to nightly in next commit)